lib/doctest.mjs

```diff
@@ -0,0 +1,62 @@
+import {readFile, writeFile, unlink} from 'fs';
+import {resolve} from 'path';
+import {promisify} from 'util';
+
+import common from './common';
+import doctest from './doctest.js';
+
+export default async function(path, options) {
```

Comment:
Let's include a second empty line after the last import.

Reply:
:zap: